CHANGELOG.md

```diff
@@ -83,6 +83,7 @@ in the naming of release branches.
 - Remove model test framework #3019
 - The `Cardano.Ledger.Alonzo.Scripts` module no longer re-exports the
   `plutus-ledger-api`'s `assertWellFormedCostModelParams`. #3065
+- Removed unused `Data.BiMap` module from `cardano-data #3089
```

Comment:
```suggestion
- Removed unused `Data.BiMap` module from `cardano-data` #3089
```

Reply:
uh oh, we've "ticked" @lehins off. :drum:

Reply:
good catch, it would have annoyed me to no end to see if after i merged, thanks